docs/tracking.md

```diff
@@ -62,7 +62,7 @@ please open an issue on [this issue tracker](https://github.com/Quansight-Labs/f
 | numexpr              |       [:simple-github:](https://github.com/pydata/numexpr/issues/503)        | :material-check-bold: |                            |                            |                                                            |                                                                                              |
 | NumPy                |        [:simple-github:](https://github.com/numpy/numpy/issues/26157)        | :material-check-bold: |   :material-check-bold:    |           2.1.0            | :simple-linux: :simple-apple: :material-microsoft-windows: |      [:simple-anaconda:](https://anaconda.org/scientific-python-nightly-wheels/numpy/)       |
 | nvImageCodec         |                                                                              | :material-check-bold: |   :material-check-bold:    |           0.4.0            |                                                            |                                                                                              |
-| ONNX                 |         [:simple-github:](https://github.com/onnx/onnx/issues/6594)          | :material-check-bold: |                            |                            |                                                            |                                                                                              |
+| ONNX                 |         [:simple-github:](https://github.com/onnx/onnx/issues/6594)          | :material-check-bold: |   :material-check-bold:    |           1.18.0           |        :simple-apple: :material-microsoft-windows:         |                                                                                              |
```

Comment:
```suggestion
| ONNX                 |         [:simple-github:](https://github.com/onnx/onnx/issues/6594)          | :material-check-bold: |   :material-check-bold:    |           1.18.0           |                                                            |                                                                                              |
```

The platform icons are to indicate that there are nightly wheels available. We don't have a column for platform support on pypi.

Reply:
If onnx does have wheels though, please add the link for the last column :)